src/DotNetWorker.Grpc/Http/GrpcHttpRequestData.cs

```diff
@@ -127,7 +127,7 @@ public override HttpResponseData CreateResponse()
             return new GrpcHttpResponseData(FunctionContext, System.Net.HttpStatusCode.OK);
         }
 
-#if NET5_0
+#if NET5_0_OR_GREATER
```

Comment:
Should this change be done on `main`, outside of this PR?

Reply:
Yes. I will create a PR to fix that in main.

Reply:
PR to fix this in main: https://github.com/Azure/azure-functions-dotnet-worker/pull/1662